### DISCUSSION TITLE
Ideal approach for Navbar implementation with active and inactive state nav items [Web] ?

### DISCUSSION THREAD

**Discussion by nasiko-satya** [FilledStacks/stacked#1015] [Q&A]
What should be the ideal approach for Navbar implementation for a static website using Stacked?

Navbar's items will have active and inactive customization.

**Comment by ferrarafer** [comment]
Hi @arithmic-satya , I recommend [this](https://www.filledstacks.com/post/bottom-navigation-with-stacked-architecture/) tutorial. Let me know if you still have doubts.
